main.tex

```diff
@@ -180,6 +181,117 @@ \section*{Introduction}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section*{Methods}
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Data collection and processing}
+
+
+%
+For detailed information on Ag1000g WGS sample collection, sequencing, variant calling, quality control and phasing see Ag1000g Consortium \emph{et al}. (\cite{Ag1000gConsortium2017}).
+%
+In brief, \emph{An. gambiae} and \emph{An. coluzzii} mosquitoes were collected from eight countries across Sub-Saharan Africa: Angola, Burkina Faso, Cameroon, Gabon, Guinea, Guinea Bissau, Kenya and Uganda. 
+%
+Guinea Bissau was an \emph{An. gambiae}/\emph{coluzzii} admixed population, from Angola just \emph{An. coluzzii} were sampled, Burkina Faso had samples of both \emph{An. gambiae} and \emph{An. coluzzii} and all other populations consisted of purely \emph{An. gambiae}.
```

Comment:
In the consortial paper I took the line that for Guinea-Bissau and Kenya the species status is uncertain, as they appeared to carry a mixture of gambiae and coluzzii alleles. Saying they are "admixed" is a specific interpretation which may not be true.